acacia/env_to_config_override.py

```diff
@@ -0,0 +1,17 @@
+import os
+from ConfigParser import ConfigParser
+
+if __name__ == '__main__':
+
+    with open('/app/config', 'a') as ofile:
+        ofile.write('\n[override]\n')
+
+    config = ConfigParser()
+
+    with open('/app/config') as fp:
+        config.readfp(fp)
+
+    for par, val in sorted(os.environ.items()):
+        if par in config.options('override'):
```

Comment:
I think this will check to see if 'par' exists in the [options] section of the config file.  but the [options] section of the config is being created 10 lines up. 

should this be a not in?